### DISCUSSION TITLE
Reserved keywords

### DISCUSSION THREAD

**Discussion by PhillypHenning** [bitovi/bitops#226] [General]
We will also need to clearly point out that some attributes are not usable as parameter names.
Those being; 
["export_env", "default", "enabled", "type", "parameter", "required", "dash_type"]

Which have been reserved by BitOps

ToDo
- Create page in wiki
- Move the reserved keys out of code and into a configuration option
- Add to README

**Comment by mickmcgrath13** [comment]
@PhillypHenning this feels like it's good to convert to an issue.

**Reply by PhillypHenning** [reply]
Created an issue from the discussion
https://github.com/bitovi/bitops/issues/235
